vaccine_feed_ingest/runners/nc/myspot_gov/normalize.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+
+from vaccine_feed_ingest.schema import schema  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+
+def _get_id(site: dict) -> str:
+    loc_id = site["Event Location Id"]
+    return f"nc:myspot_gov:{loc_id}"
+
+
+def _get_name(site: dict) -> str:
+    return site["Provider Location Name"]
+
+
+def _get_address(site: dict):
+    return schema.Address(
+        street1=site["Street Address"],
+        street2=site["Street Address 2"],
+        city=site["City"],
+        state=site["State"],
+        zip=site["Postal Code"],
+    )
+
+
+def _get_location(site: dict):
+    if site["latitude"] == "" or site["longitude"] == "":
+        return None
+    return schema.LatLng(
+        latitude=float(site["latitude"]),
+        longitude=float(site["longitude"]),
+    )
+
+
+def _get_contacts(site: dict):
+    return [
+        schema.Contact(
+            phone=site["Appointment Phone"],
+            website=site["Web Address"],
+        )
```

Comment:
Could you please make the following tweaks:
* add a _separate_ contact entry for each type (so if a location has both a website and a phone, they should have _two_ contact records).
* Format the phone number like `(###) ###-####` ([see schema](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Normalized-Location-Schema#specification))

Reply:
On it; should be able to address in ~8 hours.

Reply:
Did this.

Some phone numbers had additional info (like "extension 2" or "option 1"); I put this in the "other" field of the same contact entry, prepended with "phone_notes:". Let me know if we'd prefer something else?

Reply:
That seems reasonable - our schema does not have a good way to handle options/extensions. CC: @bryanculbertson for potential modifications.